src/components/Wallet/WalletSpend.jsx

```diff
@@ -194,7 +157,17 @@ class WalletSpend extends React.Component {
               <Grid item md={12}>
                 <Grid container direction="row-reverse">
                   <Box display="flex-end">
-                    <Box p={1}>{this.renderSpend()}</Box>
```

Comment:
no reason this needs to be a separate method, so I moved it into the main render function body.